eclair-core/src/test/scala/fr/acinq/eclair/payment/Bolt11InvoiceSpec.scala

```diff
@@ -575,4 +576,12 @@ class Bolt11InvoiceSpec extends AnyFunSuite {
       assert(Bolt11Invoice.fromString(req).toString == req)
     }
   }
+
+  test("large unknown feature"){
```

Comment:
This test should also serialize and deserialize the invoice and verify that we get the same result.

Can you also add a separate test that showcases the exception we get if we try to serialize an invoice that contains a feature bit that's way too big?

Reply:
Done